### PR TITLE
Specify 1.3 release when installing with helm

### PR DESCRIPTION
**Pull request by pmalek-sumo**
###### Description

Specify `--version` flag when installing with helm so that users don't get surprised by getting e.g. 2.0 (with breaking changes) installed.

Addresses https://github.com/SumoLogic/sumologic-kubernetes-collection/issues/1502

### CODE REVIEW

**Review by sumo-drosiek — APPROVED** [review]

**Review by perk-sumo — COMMENTED** [review]
👍

**Comment by astencel-sumo** [comment]
This should have been two separate commits - one with reformatting, another one with the change. It's all but impossible to see where the actual change is among the formatting changes.

File: deploy/docs/Installation_with_Helm.md [hunk 1]
```diff
@@ -98,51 +128,74 @@ Once you have completed installation, you can [install the Kubernetes App and vi
...
 ```
 
-Next, you can run `helm upgrade --install` to upgrade to that version. The following upgrades the current version of `my-release` to the latest.
+Next, you can run `helm upgrade --install` to upgrade to that version.
+
+The following command upgrades the current version of `my-release` to the latest `1.x` release.
 
 ```bash
-helm upgrade --install my-release sumologic/sumologic -f values.yaml
+helm upgrade --install my-release sumologic/sumologic -f values.yaml --version=1
```
Comment (astencel-sumo) [inline deploy/docs/Installation_with_Helm.md:198]: Why is it just `1` here and `1.3` above in line 177?

Comment (astencel-sumo) [reply deploy/docs/Installation_with_Helm.md:198]: Oh nevermind I see now, the added line above says "to upgrade to latest 1.x"

**Pull request merged**